docs/writing-pipelines.md

```diff
@@ -35,19 +35,18 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 nameSuffix: -go-pipeline
 resources:
-  - ../../../components/base/pipeline.yaml
-  - ../../../components/base/task.yaml
+  - /components/base/pipeline.yaml
+  - /components/base/task.yaml
 components:
-  - ../../../components/sources/git
-  - ../../../components/producers/aggregator
-  - ../../../components/producers/golang-gosec
-  - ../../../components/producers/golang-nancy
-  - ../../../components/enrichers/aggregator
-  - ../../../components/enrichers/policy
-  - ../../../components/enrichers/deduplication
-  - ../../../components/consumers/mongodb
-  - ../../../components/consumers/elasticsearch
-
+  - /components/sources/git
```

Comment:
wont' the `/` in the beginning cause an issue when the path is merged?

Reply:
When tested locally while building the atom enricher this did the trick :)